### PR TITLE
ENGG-2124: Show tooltip for large rule and mock names in listing view

### PR DESCRIPTION
**Pull request by RuntimeTerror10**

### CODE REVIEW

File: app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/rulesTable.css [hunk 1]
```diff
@@ -104,26 +104,43 @@
...
+.rule-name-container .rule-name .ant-typography {
+  /* TODO: use theme variables */
+  color: #3ba0ff;
+  transition: color 0.2s ease;
+}
+
+.rule-name-container .rule-name .ant-typography:hover {
+  /* TODO: use theme variables */
+  color: #1f50b1;
+}
```
Comment (rohanmathur91) [inline app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/rulesTable.css:116]: Why we didn't used variables?

File: app/src/features/mocks/screens/mocksList/components/MocksList/components/MocksTable/mocksTable.scss [hunk 2]
```diff
@@ -10,6 +10,11 @@
   padding-left: 10px !important;
 }
 
+.mocks-table-ellipsis-tooltip .ant-tooltip-inner {
+  font-size: var(--requestly-font-size-sm);
+  background: var(--requestly-color-black);
+}
```
Comment (rohanmathur91) [inline app/src/features/mocks/screens/mocksList/components/MocksList/components/MocksTable/mocksTable.scss:16]: Possible to keep this styles at common level?

**Review by rohanmathur91 — CHANGES_REQUESTED** [review]
Is it possible to handle the ellipsis in content table itself?

File: app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/rulesTable.css (hunk 1 above)
Comment (RuntimeTerror10) [reply app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/rulesTable.css:116]: The rule names in rules table are links whose styles are being applied from top level, and these colors are not present in our tokens.

**Comment by RuntimeTerror10** [comment]
> Is it possible to handle the ellipsis in content table itself?

Tried the Table's ellipsis property but behaviour is different from the requirements

**Review by rohanmathur91 — CHANGES_REQUESTED** [review]

**Review by rohanmathur91 — APPROVED** [review]

**Pull request merged**